src/main/java/org/amahi/anywhere/adapter/NavigationDrawerAdapter.java

```diff
@@ -1,32 +1,49 @@
 package org.amahi.anywhere.adapter;
 
 import android.content.Context;
+import android.graphics.drawable.Drawable;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.TextView;
 
-import org.amahi.anywhere.R;
+import com.amulyakhare.textdrawable.TextDrawable;
+import com.amulyakhare.textdrawable.util.ColorGenerator;
 
+import org.amahi.anywhere.R;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
+
 /**
  * Navigation Drawer adapter. Visualizes predefined values
  * for the {@link org.amahi.anywhere.fragment.NavigationFragment}.
  */
 
 public class NavigationDrawerAdapter extends RecyclerView.Adapter<NavigationDrawerAdapter.NavigationDrawerViewHolder> {
 
-    private final List<Integer> navigationItems;
+    private final List<Integer> mNavigationItems;
     private Context mContext;
 
-    public NavigationDrawerAdapter(Context context, List<Integer> navigationItems) {
-        this.navigationItems = navigationItems;
+    private final ArrayList<String> mServerName;
+
+
+    //for Old Navigation List
+    private NavigationDrawerAdapter(Context context, List<Integer> navigationItems) {
```

Comment:
do we need it?

Reply:
Array of String is for **server name** list. List of Integer is for **Selected Server Option (share,apps,etc)**, made by someone before.